src/utils/getMetricValueWithFilters/getMetricValueWithFilters.ts

```diff
@@ -0,0 +1,110 @@
+import { DataFrame, Field, PanelData } from "@grafana/data";
+
+import { getShowcaseMetricValue } from "../metricValue/getShowcaseMetricValue";
+
+declare const data: PanelData;
+
+function getMultipleSeriesFromName(
+  seriesName: string
+): DataFrame[] | undefined {
```

Comment:
This doesn't return undefined
```suggestion
): DataFrame[] {
```